simple-stack-example-kotlin-dagger/src/main/java/com/zhuinden/simplestackkotlindaggerexample/schedulers/SchedulerProviderImpl.kt

```diff
@@ -0,0 +1,20 @@
+package com.zhuinden.simplestackkotlindaggerexample.schedulers
+
+import io.reactivex.Scheduler
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+class SchedulerProviderImpl : SchedulerProvider {
+
+    override fun computation(): Scheduler {
+        return Schedulers.computation()
```

Comment:
I'd use `=` here instead of `return` but that is a matter of style at this point :smile: